lib/elixir_console_web/live/console_live/history_component.html.leex

```diff
@@ -1,7 +1,10 @@
+<% alias ElixirConsoleWeb.ConsoleLive.HistoryComponent  %>
+<% alias ElixirConsoleWeb.ConsoleLive %>
+
 <div class="flex-1"></div>
 <div class="p-2" id="commandOutput" phx-update="append">
   <%= for output <- @output do %>
-    <div id="command<%= output.id %>" class="text-gray-300 font-medium"><%= print_prompt() %><%= format_command(output.command) %></div>
```

Comment:
`<%= format_command(output.command) %>` just works.. the corresponding module is "available" from here.

(it is right that `print_prompt` does not work because this function is not from `HistoryComponent `

As a consequence, the `alias ... HistoryComponent` is not needed.

Reply:
Excellent!